doc/version_history.rst

```diff
@@ -17,6 +17,16 @@ Version History
 
 .. towncrier release notes start
 
+.. _lsst.ts.wep-16.2.1:
```

Comment:
This is not the way of updating the version history any more. Please read the WARNING above